app/services/fetch_atom_service.rb

```diff
@@ -26,27 +18,55 @@ def call(url)
 
   private
 
-  def process_html(body)
-    Rails.logger.debug 'Processing HTML'
+  def perform_request
+    @response = Request.new(:get, @url).perform
```

Comment:
doesn't this need to have `Accept: activity+json, application/ld+json; profile="whatever"` to actually get an activitypub resource?